demo/iwslt_blaser_eval/README.md

```diff
@@ -0,0 +1,87 @@
+
+# Getting started with mining
+
+Welcome to `stopes`, this is a quickstart guide to discover how to run automated pipelines with `stopes`. In this example we describe how to use BLASER to evaluate speech translation as described in the https://iwslt.org/2023/s2s task.
+
+## Installation
+
+We recommend installing all this in a dedicated conda environment to not overwrite your current python install:
+
+```bash
+# create conda env
+conda create -n blaser python=3.9
+conda activate blaser
+
+# upgrade pip
+python -m pip install --upgrade pip
+
+# install fairseq
+pip install fairseq==0.12.1
+
+# install stopes+blaser 
+pip install 'stopes[blaser]'
+```
+
+# Using BLASER
+
+The main BLASER readme has all the details of how the pipeline works: https://github.com/facebookresearch/stopes/blob/main/stopes/eval/blaser/README.md
+
+In this tutorial we'll check how this works for the IWSLT'23 speech translation track.
+
+## Getting the Data and Encoders
+
+IWSLT'23 provides some test data, we'll also need the LASER encoders for the languages in the track (English and Chinese). You can run `prepare.sh` to download the data. Run this script from the `stopes/demo/iwslt_blaser_eval` folder or adapt it to your need.
+
+```bash
+./prepare.sh
+```
+
+The script will have down three things:
+
+- download the English eval data into data/en
+- download the English and Mandarin encoders in encoders/ (this will take a while as they are big)
+- download the BLASER model and config
+- create a "manifest" file that lists all the english files to use as source
+
+## Create/Prepare your translation data
+
+Once you are ready to test your system, create one wav file per source audio file and use the same name as the input file (number.wav). Create a manifest file similar to the one created by prepare.sh, you can do this with:
+
+```bash
+python mk_manifest.py manifest -i yourdatadir > yourmanifestpath.tsv
+```
+
+You should also create such manifest for your reference audio files.
+
+## You can now run blaser eval with:
+
+A lot of config is preset in `conf/eval_blaser.yaml`, please see the comments in that configuration
+file if you need to adjust anything. After that you can run the following command:
+
+```bash
+DEMO_DIR=<ENTERPATHTOIWSLTBLASEREVALDIRECTORY> python -m stopes.pipelines.eval.eval_blaser --config-dir $DEMO_DIR/conf tgt_manifest=<PATHTOTARGETMANIFEST.tsv> ref_manifest=<PATHTOREFERENCEMANIFEST.tsv>
+```
+
+Make sure to replace:
+
+- `ENTERPATHTOIWSLTBLASEREVALDIRECTORY` to the directory where you've run prepare.sh and downloaded everything
```

Comment:
are we forced to use these filenames without underscores?

Reply:
no, it's just a placeholder :)